crates/store/src/backend/tikv/blob.rs

```diff
@@ -0,0 +1,167 @@
+/*
+ * SPDX-FileCopyrightText: 2020 Stalwart Labs Ltd <hello@stalw.art>
```

Comment:
```suggestion
 * SPDX-FileCopyrightText: 2024 Stalwart Labs Ltd <hello@stalw.art>
```
Should it not start at file creation ?

Reply:
You're right. Alongside the years, I just copypasted the entire FoundationDB implementation at first, before realising that they have totally different limitations and needed rewrite lol